cmd/engine/engine.go

```diff
@@ -44,11 +44,13 @@ import (
 	"clouditor.io/clouditor/api/assessment"
 	"clouditor.io/clouditor/api/auth"
 	"clouditor.io/clouditor/api/discovery"
+	"clouditor.io/clouditor/api/evidence_store"
```

Comment:
According to [Link](https://golang.org/doc/effective_go#package-names) the package names shouldn´t be in camelCase or snake_case.